app/src/main/java/com/github/kulebin/myfoursquareapp/IView.java

```diff
@@ -0,0 +1,9 @@
+package com.github.kulebin.myfoursquareapp;
+
+public interface IView {
```

Comment:
Name should tell what for this interface. It's about result, status, or it delegates something like this